src/Resources/skeleton/forgottenPassword/twig_request.tpl.php

```diff
@@ -0,0 +1,11 @@
+<?= $helper->getHeadPrintCode('Recover your password'); ?>
+
+{% block body %}
+    <h1>Recover your password</h1>
+
+    {{ form_start(requestForm) }}
+        {{ form_row(requestForm.<?= $email_field ?>) }}
```

Comment:
I have mixed feeling there. Result will be the same, but it will be easier for developers customize the label with `form_row`. If we use `form_widget`, it may create a trap for newcomers who will modify the form label or the form theme.

Reply:
Ah, I see... I almost always do everything in form types. Anyway, it's minor and as far as it works the same - I'm fine with it 👍